src/main/java/uk/gov/hmcts/divorce/citizen/event/CitizenSwitchedToSoleCo.java

```diff
@@ -95,6 +101,11 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(CaseDetails<C
             switchToSoleService.switchApplicantData(data);
         }
 
+        generateConditionalOrderAnswersDocument.apply(
+            details,
+            isApplicant1 ? data.getApplicant1().getLanguagePreference() : data.getApplicant2().getLanguagePreference()
+        );
```

Comment:
I think there is another issue here with the language preference, as the applicants may have been switched on line 101